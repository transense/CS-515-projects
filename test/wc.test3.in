hi
bye

hello how are you ?
