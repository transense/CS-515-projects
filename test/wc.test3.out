     4   7   28 test/wc.test3.in